cves/CVE-2009-3548.yml

```diff
@@ -116,9 +123,12 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
+  answer: |
+    This issue was reported directly to the tomcat users public mailing
+    list by David Horheim. Link - 
+    https://markmail.org/thread/wfu4nff5chvkb6xp#query:+page:1+mid:bva3jqihjxscatev+state:results
```

Comment:
Does it say specifically how this vulnerability was found?

Reply:
He mentioned that while downloading exe version of the installer (specifically not zip or tar.gz) it enables the manager application - which leads to the default blank password.